lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -76,10 +76,12 @@ def script
             local swift_runtime_libs
             swift_runtime_libs=$(xcrun otool -LX "${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}/$1/${basename}" | grep @rpath/libswift | sed -E s/@rpath\\\\/\\(.+dylib\\).*/\\\\1/g | uniq -u  && exit ${PIPESTATUS[0]})
             for lib in $swift_runtime_libs; do
-              echo "rsync -av \\"${SWIFT_STDLIB_PATH}/${lib}\\" \\"${destination}\\""
-              rsync -av "${SWIFT_STDLIB_PATH}/${lib}" "${destination}"
-              if [ "${CODE_SIGNING_REQUIRED}" == "YES" ]; then
-                code_sign "${destination}/${lib}"
+              if [ -n "${destination}" ]; then
```

Comment:
Here's a good resource for bash magic: [![screen shot 2015-02-17 at 16 50 47](https://cloud.githubusercontent.com/assets/44164/6232618/242e6a2c-b6c5-11e4-9354-71504dd6c4a7.png)](http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html)